packages/shared/src/clients/workflows-client.ts

```diff
@@ -98,5 +107,6 @@ export const createWorkflowClient = (baseUrl: string, headers: HeadersInit = {})
     searchWorkflows,
     getWorkflowTestData,
     getWorkflowStepData,
+    patchWorkflowStepData, // Include the new method in the returned object
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Seems like a redundant/superfluous, was this intentional?‏
```suggestion
    patchWorkflowStepData,
```

Reply:
chatgpt... forgot to tell him no comments